[2025-07-22T09:07:04.690+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-22T09:07:04.736+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tunestream_etl.stage_songs manual__2025-07-22T09:01:45.344716+00:00 [queued]>
[2025-07-22T09:07:04.750+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tunestream_etl.stage_songs manual__2025-07-22T09:01:45.344716+00:00 [queued]>
[2025-07-22T09:07:04.751+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 4
[2025-07-22T09:07:04.782+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): stage_songs> on 2025-07-22 09:01:45.344716+00:00
[2025-07-22T09:07:04.794+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=7409) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-22T09:07:04.795+0000] {standard_task_runner.py:64} INFO - Started process 7410 to run task
[2025-07-22T09:07:04.794+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tunestream_etl', 'stage_songs', 'manual__2025-07-22T09:01:45.344716+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/tunestream_etl_dag.py', '--cfg-path', '/tmp/tmpbgqv0rhw']
[2025-07-22T09:07:04.796+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask stage_songs
[2025-07-22T09:07:04.820+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-22T09:07:04.880+0000] {task_command.py:426} INFO - Running <TaskInstance: tunestream_etl.stage_songs manual__2025-07-22T09:01:45.344716+00:00 [running]> on host 7b3eb14378b0
[2025-07-22T09:07:05.065+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='NamTH91' AIRFLOW_CTX_DAG_ID='tunestream_etl' AIRFLOW_CTX_TASK_ID='stage_songs' AIRFLOW_CTX_EXECUTION_DATE='2025-07-22T09:01:45.344716+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-22T09:01:45.344716+00:00'
[2025-07-22T09:07:05.066+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-22T09:07:05.103+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-22T09:07:05.104+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/tunestream_etl_dag.py", line 27, in load_song_data_to_table
    print(f"tổng số file json tìm được là {len(all_files)}")
                                               ^^^^^^^^^
NameError: name 'all_files' is not defined
[2025-07-22T09:07:05.188+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=tunestream_etl, task_id=stage_songs, run_id=manual__2025-07-22T09:01:45.344716+00:00, execution_date=20250722T090145, start_date=20250722T090704, end_date=20250722T090705
[2025-07-22T09:07:05.232+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 14 for task stage_songs (name 'all_files' is not defined; 7410)
[2025-07-22T09:07:05.297+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2025-07-22T09:07:05.368+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-22T09:07:05.376+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
